tests/sls/homu/valid_repos.py

```diff
@@ -0,0 +1,33 @@
+import toml
+import urllib.request
+from urllib.error import URLError
+from tests.util import Failure, Success
+
+
+def getStatus(url):
+    '''
+    Consumes a url string and returns the status code of a GET request
+    '''
+    try:
+        response = urllib.request.urlopen(url).status
```

Comment:
Use HEAD requests to reduce bandwidth usage - you can create a `urllib.request.Request` object with `method='HEAD'`, and pass that to `urlopen`.
Use a with statement for `urlopen` to automatically close the response.